// Copyright 2020 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

package feedwire;

option optimize_for = LITE_RUNTIME;

enum Capability {
  UNKNOWN_CAPABILITY = 0;
  INFINITE_FEED = 5;
  DISMISS_COMMAND = 9;
  UNDO_FOR_DISMISS_COMMAND = 10;
  SPORTS_IN_GAME_UPDATE = 35;
  PERSIST_GAME_CONTENT = 46;
  OPEN_VIDEO_COMMAND = 16;
  UI_THEME_V2 = 17;
  INLINE_VIDEO_AUTOPLAY = 18;
  CARD_MENU = 19;
  REQUEST_SCHEDULE = 20;
  HEART = 23;
  SHARE = 24;
  OPEN_IN_TAB = 27;
  DOWNLOAD_LINK = 28;
  READ_LATER = 30;
  LOTTIE_ANIMATIONS = 37;
  LONG_PRESS_CARD_MENU = 38;
  PREFETCH_METADATA = 43;
  SILK_AMP_OPEN_COMMAND = 44;
  AMP_STORY_PLAYER = 47;
  AMP_GROUP_DATASTORE = 48;
  MATERIAL_NEXT_BASELINE = 60;
  MATERIAL_NEXT_GOOGLE_SANS = 61;
  CONTENT_LIFETIME = 64;
  OPEN_IN_INCOGNITO = 67;
  ON_DEVICE_USER_PROFILE = 70;
  INVALIDATE_CACHE_COMMAND = 73;
  INFO_CARD_ACKNOWLEDGEMENT_TRACKING = 77;
  THANK_CREATOR = 79;
  OPEN_IN_NEW_TAB_IN_GROUP = 81;
  SYNTHETIC_CAPABILITIES = 89;
  OPEN_WEB_FEED_COMMAND = 98;
}
